contrib/syslog-ng-debun

```diff
@@ -649,43 +655,57 @@ acquire_syslog_pids () {
 	echo 'Old "getsyslogpids":' >${tmpdir}/syslog.pids
 	getsyslogpids >>${tmpdir}/syslog.pids
 	if [ -r "${piddir}/syslog-ng.pid" ]; then
-		sngpid=`cat ${piddir}/syslog-ng.pid`
+		sngpid=$( cat ${piddir}/syslog-ng.pid )
 	else
 		# Handle when fhs is "linux"-like
-		sngpid=`cat ${piddir}/syslog-ng.pid`
+		sngpid=$( cat ${piddir}/syslog-ng.pid )
 	fi
-	ppid=`getparent $sngpid`
-	sngallpids="`getallchilds $sngpid`"
+	ppid=$( getparent $sngpid )
+	sngallpids="$( getallchilds $sngpid )"
 	echo "SVpid: $ppid SNGpid: $sngpid Chpids: ${sngallpids}" >>${tmpdir}/syslog.pids
 	tail -1 ${tmpdir}/syslog.pids
 	if [ -n "$ppid" ]; then
 		sngallpids="$ppid $sngpid ${sngallpids}"
 	else
-		sngallpids="` getsyslogpids `"
+		sngallpids="$( getsyslogpids )"
 	fi
 	# drop out the unneeded white spaces, since that disturb the ps command
-	sngallpids=`echo ${sngallpids}`
+	sngallpids=$( echo ${sngallpids} )
 	if [ -n "${sngallpids}" ]; then
 		printf 'ps -l -f -p "%s"\n' "${sngallpids}" >>${tmpdir}/syslog.pids
 		ps -l -f -p "${sngallpids}" >>${tmpdir}/syslog.pids
 	fi
 	[ -n "$service_status" ] && $service_status >${tmpdir}/svc.pre
 }
 
-acquire_syslog_info () {
-	ls -la "${binprefix}" > ${tmpdir}/syslog.lsl.install_dir 2>&1
-	printf "Syslog-ng's exact version: "
-	$syslogbin --version > ${tmpdir}/syslog.version
-	head -1 ${tmpdir}/syslog.version
-	${syslogbin}-ctl stats > ${tmpdir}/syslog.stats 2>&1
+acquire_syslog_stats () {
+	local tsdata
+	tsdata=$( timestamp )
+	#handy-dandy delay magic, triggered when we'd step on our own foot
+	[ -f ${tmpdir}/syslog.stats.${tsdata} ] && sleep 5 && tsdata=$( timestamp )
+
+	${syslogbin}-ctl stats > ${tmpdir}/syslog.stats.${tsdata} 2>&1
```

Comment:
It would be better (for reading) to create a separate variable for ${syslogbin}-ctl it is used in a few places.